shopfloor_reception/tests/test_select_document.py

```diff
@@ -151,6 +151,7 @@ def test_scan_not_tracked_product_single_picking(self):
             data={
                 "picking": data,
                 "selected_move_line": self.data.move_lines(selected_move_line),
+                "confirmation_required": False,
```

Comment:
move the generation of this data to an helper method and avoid changing it everywhere again ;)